### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tavisca-snarhe**
done change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Search
```
Comment (tavisca-snarhe) [inline README.md]: snarhe - 1

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+# Search
```
Comment (tavisca-akhullar) [inline README.md]: akhulllar -1

Comment (tavisca-akhullar) [inline README.md]: akhullar - 2

File: README.md (hunk 1 above)
Comment (tavisca-akhullar) [reply README.md]: akhullar - rpy - 3

Comment (tavisca-snarhe) [reply README.md]: snarhe - hereyougo - 2

**Pull request merged**